test/unit/player.test.js

```diff
@@ -172,7 +172,6 @@ test('should set the width, height, and aspect ratio via a css class', function(
   };
 
   // Initial state
-  ok(player.styleEl_.parentNode === player.el(), 'player has a style element');
```

Comment:
we're no longer adding this element to the player. The tests below should cover whether the styles are applied correctly or not.